fastcrud/endpoint/endpoint_creator.py

```diff
@@ -1,24 +1,47 @@
+import inspect
 from typing import Type, TypeVar, Optional, Callable, Sequence, Union
 from enum import Enum
 
 from fastapi import Depends, Body, Query, APIRouter, params
+from pydantic import BaseModel, ValidationError
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.orm import DeclarativeBase
-from pydantic import BaseModel, ValidationError
 
-from ..exceptions.http_exceptions import NotFoundException
 from ..crud.fast_crud import FastCRUD
-from ..exceptions.http_exceptions import DuplicateValueException
-from .helper import CRUDMethods, _get_primary_key, _extract_unique_columns
-from ..paginated.response import paginated_response
+from ..exceptions.http_exceptions import DuplicateValueException, NotFoundException
 from ..paginated.helper import compute_offset
+from ..paginated.response import paginated_response
+from .helper import CRUDMethods, _extract_unique_columns, _get_primary_keys
 
 CreateSchemaType = TypeVar("CreateSchemaType", bound=BaseModel)
 UpdateSchemaType = TypeVar("UpdateSchemaType", bound=BaseModel)
 UpdateSchemaInternalType = TypeVar("UpdateSchemaInternalType", bound=BaseModel)
 DeleteSchemaType = TypeVar("DeleteSchemaType", bound=BaseModel)
 
 
+def apply_model_pk(**pkeys):
```

Comment:
Could you add type hints and a short description here? The code itself is self-documenting, but just to make it even easier for other people